dapps/shop/backend/queues/queues.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file Queue objects for adding processing dshop jobs.
+ *
+ * If no REDIS_URL defined, will fallback to prossing jobs as the jobs are
+ * submitted, without using a queue.
+ */
+
+const { REDIS_URL } = require('../utils/const')
+
+const Queue = REDIS_URL ? require('bullmq') : require('./fallbackQueue')
+const backendUrl = REDIS_URL ? REDIS_URL : undefined
+const queueOpts = {}
+
+const all = [
+  new Queue('makeOffer', backendUrl, queueOpts),
+  new Queue('download', backendUrl, queueOpts),
+  new Queue('discord', backendUrl, queueOpts),
+  new Queue(
+    'email',
+    backendUrl,
+    Object.assign(queueOpts, {
+      maxStalledCount: 0 // We don't want to risk sending an email twice
+    })
+  )
```

Comment:
Are these other queues just setup for later?  I don't see anything attaching to them or adding to them.

Reply:
Yes. Going to be the next thing I code on this.